components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -126,25 +139,41 @@ public void addOAuthApplication(OAuthAppDO consumerAppDO) throws IdentityOAuthAd
                         prepStmt.setLong(11, consumerAppDO.getApplicationAccessTokenExpiryTime());
                         prepStmt.setLong(12, consumerAppDO.getRefreshTokenExpiryTime());
                         prepStmt.execute();
+                        try (ResultSet results = prepStmt.getGeneratedKeys()) {
+                            if (results.next()) {
+                                appId = results.getInt(1);
+                            }
+                        }
                     }
                 }
-
+                // some JDBC Drivers returns this in the result, some don't
```

Comment:
Comments should start with capital letter.

Reply:
fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/942/commits/ca4690d734709b108ef2df33a69ae3c9242febc8